producer/csc/test_client.py

```diff
@@ -19,7 +20,7 @@ async def test_valid_remote_not_in_config(self):
         async def arrange():
             from csc.client import CSCWSClient
 
-            salobj.set_random_lsst_dds_domain()
+            salobj.set_random_lsst_dds_partition_prefix()
```

Comment:
What is the difference between this line and the line of the before version?

Reply:
The `set_random_lsst_dds_domain` was deprecated and replaced for `set_random_lsst_dds_partition_prefix`. Here is the documentation link: https://ts-salobj.lsst.io/py-api/lsst.ts.salobj.set_random_lsst_dds_domain.html

Reply:
Great! ;)